lib/peep.rb

```diff
@@ -0,0 +1,16 @@
+class Peep
+
+  include DataMapper::Resource
+
+  property :id,                Serial
+  property :name,              String
+  property :user_name,         String
+  property :created_at,        DateTime
+
+  belongs_to :user
+  has n, :peeps, through: Resource
```

Comment:
Not following now - Peep has associations to peeps, is that what you are referring to?